public/utils/helpers.tsx

```diff
@@ -165,14 +166,65 @@ export function renderVisualization(spec: TopLevelSpec, containerId: string) {
   }
 
   function renderVegaSpec(spec: {}) {
-    const handler = new Handler();
+    let pathGroup: any[] = [];
```

Comment:
Doubt: Is `pathGroup` a term specific to Vega tooltip? If not can we name it something that describes what it means?

Reply:
It is specific to charts in general, as charts consist of a bunch of paths, and here I filter only the paths that contain the colors to update the tooltip. The path is a general name for chart elements.
Something more intuitive perhaps: chartColoredItems or chartColors or tooltipColors?

Reply:
@amsiglan any thoughts?

Reply:
`chartColoredItems` seems apt here

Reply:
Updated.